config/config.go

```diff
@@ -36,6 +42,12 @@ func Get() (*Config, error) {
 		GracefulShutdownTimeout:    5 * time.Second,
 		HealthCheckInterval:        30 * time.Second,
 		HealthCheckCriticalTimeout: 90 * time.Second,
+		EnablePrivateEndpoints:     true,
+		ZebedeeURL:                 "http://localhost:8082",
+		ServiceAuthToken:           "",
+		EnablePermissionsAuth:      false,
```

Comment:
Probably better to have permissionsAuth default to true?

Reply:
I would err on the side of caution and leave as it. I think people are instructed in general to set ENABLE_PERMISSIONS_AUTH=true in their rc files, and this interferes with component tests. I have tried setting `os.clearenv()` in the `main_test.go` file, to no avail, so would keep as is, unless you know a way around that